src/primitives/element.js

```diff
@@ -188,6 +188,7 @@ class Element {
       elements = elements.filter((element) => {
         let parentElements = element.parents.map(e => e.element);
 
+        // eslint-disable-next-line no-restricted-syntax
```

Comment:
why?

Reply:
oh I see, it returns from the iteration... okay then, this would require larger refactoring I guess